include/contractor/contractor.hpp

```diff
@@ -89,7 +89,8 @@ class Contractor
                           const std::string &geometry_filename,
                           const std::string &datasource_names_filename,
                           const std::string &datasource_indexes_filename,
-                          const std::string &rtree_leaf_filename);
+                          const std::string &rtree_leaf_filename,
+                          const double &verify_weights);
```

Comment:
> Also, can we add a default parameter value to disable checks if not wanted?

As for this, is it enough that the default value is `0` and we only apply this check if the value is greater than `0`?